windows/src/global/delphi/ui/XMLRenderer.pas

```diff
@@ -215,7 +215,6 @@ function TXMLRenderers.RenderToString(FRefreshKeyman: Boolean; const AdditionalD
         IfThen(CanElevate, '<canelevate />')+
         IfThen(Assigned(kmcom) and kmcom.SystemInfo.IsAdministrator, '<isadmin />')+   // I3612   // I3626
         '<uilanguages>'+GetUILanguages+'</uilanguages>'+
-        '<defaultlocalepath>'+XMLEncode(FTemplatePath)+'locale.xml</defaultlocalepath>'+
         '<localepath>'+XMLEncode(s)+'</localepath>');
```

Comment:
Checking `<defaultlocalepath>` was no longer needed

Reply:
Correct -- it's now always `strings.xml` in the same path as the template, so no path information needed (in the past this was really messy, but improved with internal web server).